apps/web/src/pages/templates/components/VariantsPage.tsx

```diff
@@ -65,6 +203,9 @@ export function VariantsPage() {
         onScrollPositionChange={setScrollPosition}
       >
         {variants?.map((variant, variantIndex) => {
+          const isActiveError = errorState?.variantIndex === variantIndex;
```

Comment:
if the error navigations is currently showing an error from this particular variant, then having it `true` helps in the scroll into view and applying error border